### DISCUSSION TITLE
how to solve this error "WARN[0000] The "DD_DATABASE_NAME" variable is not set. Defaulting to a blank string."

### DISCUSSION THREAD

**Discussion by tx-abhay** [DefectDojo/django-DefectDojo#6107] [General]
![image](https://user-images.githubusercontent.com/100664313/161206213-e2fcac5f-bd33-40a6-9d3a-1e7a5cb3f7fb.png)

**Comment by tx-abhay** [comment]
@dougmorato @Juu @hasantayyar @fabaff  can u please help me out

**Comment by valentijnscholten** [comment]
What happens if you follow the docs? 

![image](https://user-images.githubusercontent.com/4426050/161278419-d1a7ff8f-569f-4e3d-b98b-d690a943275d.png)


https://defectdojo.github.io/django-DefectDojo/getting_started/installation/
https://github.com/DefectDojo/django-DefectDojo/blob/dev/readme-docs/DOCKER.md

**Comment by tx-abhay** [comment]
![image](https://user-images.githubusercontent.com/100664313/161513193-8aaf5a2a-8764-490e-beec-e301de805c92.png)

 still facing this issue

**Reply by dsever** [reply]
What about `./dc-up.sh mysql-rabbitmq` ?

or 

`docker-compose --profile mysql-rabbitmt --env-file ./docker/environments/mysql-rabbitmq.env up --no-deps -d`

**Reply by tx-abhay** [reply]
![image](https://user-images.githubusercontent.com/100664313/161516982-792fc03d-59d8-4742-b274-b203171eccc7.png)

**Reply by valentijnscholten** [reply]
The docs state you should use `./dc-up.sh --profile mysql-rabbitmq`

EDIT: should be: `./dc-up.sh --env-file mysql-rabbitmq` if I read it correctly.

**Reply by tx-abhay** [reply]
![image](https://user-images.githubusercontent.com/100664313/161517617-f3270949-0e49-4ad7-ae90-24023b504039.png)

**Reply by dsever** [reply]
Please tun this .`/dc-up.sh mysql-rabbitmq` there is no --profile switch in command

**Reply by valentijnscholten** [reply]
won't work, you need to specify the profile:
![image](https://user-images.githubusercontent.com/4426050/161520060-7ebab9b9-191c-4cca-a8d5-bec4afc653ac.png)

**Reply by tx-abhay** [reply]
Now its working

**Reply by tx-abhay** [reply]
@valentijnscholten  can we connect i have some extra doubt about defect dojo

**Reply by valentijnscholten** [reply]
you can discuss on slack with other users

**Reply by tx-abhay** [reply]
@valentijnscholten  can we connect on slack Please

**Comment by tx-abhay** [comment]
![image](https://user-images.githubusercontent.com/100664313/169556153-64518330-168c-442a-a507-1f20013f934f.png)

**Reply by tx-abhay** [reply]
@valentijnscholten @dsever @dougmorato @Juu
